Carbon.IIS/Functions/Get-CIisApplication.ps1

```diff
@@ -40,6 +40,7 @@ function Get-CIisApplication
 

```

Comment:
Support `-LocationPath` here, or is only for outer cmdlets?

Reply:
The `Microsoft.Web.Administration` API has specific classes for managing websites, application pools, applications, and virtual directories. Because of this, there's no need to use `LocationPath` , just use the name of the website and application and use the API to get applications under that website.